packages/command-palette/src/commands/use-single-site-commands.tsx

```diff
@@ -0,0 +1,927 @@
+			].join( ' ' ),
+			context: [ '/sites' ],
+			callback: commandNavigation( '/hosting-config/:site' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			siteType: SiteType.ATOMIC,
+			filterP2: true,
+			icon: settingsIcon,
+		},
+		{
+			name: 'openPHPmyAdmin',
+			label: __( 'Open database in phpMyAdmin', __i18n_text_domain__ ),
+			searchLabel: [
+				_x(
+					'open database in phpmyadmin',
+					'Keyword for the Open database in phpMyAdmin command',
+					__i18n_text_domain__
+				),
+				_x(
+					'database',
+					'Keyword for the Open database in phpMyAdmin command',
+					__i18n_text_domain__
+				),
+				_x( 'mysql', 'Keyword for the Open database in phpMyAdmin command', __i18n_text_domain__ ),
+				_x(
+					'phpmyadmin',
+					'Keyword for the Open database in phpMyAdmin command',
+					__i18n_text_domain__
+				),
+			].join( ' ' ),
+			context: [ '/sites' ],
+			callback: commandNavigation( '/hosting-config/:site#database-access' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			siteType: SiteType.ATOMIC,
+			icon: pageIcon,
+		},
+		{
+			name: 'openProfile',
+			label: __( 'Open my profile', __i18n_text_domain__ ),
+			searchLabel: [
+				_x( 'open my profile', 'Keyword for the Open my profile command', __i18n_text_domain__ ),
+				_x( 'account', 'Keyword for the Open my profile command', __i18n_text_domain__ ),
+				_x( 'display name', 'Keyword for the Open my profile command', __i18n_text_domain__ ),
+				_x( 'gravatar', 'Keyword for the Open my profile command', __i18n_text_domain__ ),
+			].join( ' ' ),
+			context: [ '/sites' ],
+			callback: commandNavigation( `/me` ),
+			icon: profileIcon,
+		},
+		{
+			name: 'viewDeveloperFeatures',
+			label: __( 'View developer features', __i18n_text_domain__ ),
+			searchLabel: [
+				_x(
+					'view developer features',
+					'Keyword for the View developer features command',
+					__i18n_text_domain__
+				),
+				_x( 'profile', 'Keyword for the View developer features command', __i18n_text_domain__ ),
+			].join( ' ' ),
+			callback: commandNavigation( `/me/developer` ),
+			icon: codeIcon,
+		},
+		{
+			name: 'openReader',
+			label: __( 'Open reader', __i18n_text_domain__ ),
+			callback: commandNavigation( `/read` ),
+			icon: <Gridicon icon="reader" />,
+		},
+		{
+			name: 'openJetpackSettings',
+			label: __( 'Open Jetpack settings', __i18n_text_domain__ ),
+			callback: commandNavigation( `/wp-admin/admin.php?page=jetpack#/settings` ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			siteType: SiteType.ATOMIC,
+			icon: <JetpackLogo className="gridicon" size={ 18 } />,
+		},
+		{
+			name: 'openJetpackSettings',
+			label: __( 'Open Jetpack settings', __i18n_text_domain__ ),
+			callback: commandNavigation( `INVESTIGATE IF WE NEED A SIMPLE SITE VERSION` ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			siteType: SiteType.SIMPLE,
+			icon: <JetpackLogo className="gridicon" size={ 18 } />,
+		},
```

Comment:
This command is duplicated, I guess you wanted to add the "Add Jetpack" command instead.

Reply:
I think I misunderstood this. Looks like the second command is a placeholder for simple sites. [We don't need it](https://github.com/Automattic/wp-calypso/blob/48bbe440e5bb7a6411c78ced61e82032e45a96a7/client/sites-dashboard/components/wpcom-smp-commands.tsx#L491).

The ["Add Jetpack" command](https://github.com/Automattic/wp-calypso/blob/48bbe440e5bb7a6411c78ced61e82032e45a96a7/client/sites-dashboard/components/wpcom-smp-commands.tsx#L498-L508) is still missing though.

Reply:
We can't add Jetpack to a Simple or Atomic site, they already have it.

Reply:
Added command